functions/index.js

```diff
@@ -51,6 +49,7 @@ app.get('/v1/dojo/auth/github/login', (req, res, next) => {
   });
 });
 
+app.use(mid.validateFirebaseIdToken)
```

Comment:
this will be put after the endpoint of the login due the authenticator is not required for this method